src/modules/azure.rs

```diff
@@ -0,0 +1,301 @@
+use encoding_rs::*;
+use std::fs::File;
+use std::io::{BufRead, BufReader, Read};
+use std::path::PathBuf;
+
+use super::{Context, Module, RootModuleConfig};
+
+type JValue = serde_json::Value;
+
+use crate::configs::azure::AzureConfig;
+use crate::formatter::StringFormatter;
+
+pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
+    let mut module = context.new_module("azure");
+    let config = AzureConfig::try_load(module.config);
```

Comment:
Is there some kind of environment variable to ensure this module is only shown when it's relevant? Otherwise, it should be disabled by default.

Reply:
Don't have that, so changing the module to be disabled.

Reply:
Second commit resolves this...